system/forms/system-config/admin-users.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure "Admin users" is the right category here (given we also have admin login security). Could we make this a "Frontend editing" category instead?